app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,73 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
+  ManageIQ::Providers::BaseManager::EventCatcher::Runner
+
+  include ManageIQ::Providers::Hawkular::Common::EventCatcher::RunnerMixin
+
+  TAG_URL  = 'url'.freeze
+  TAG_TYPE = 'type'.freeze
+
+  def initialize(cfg = {})
+    super
+  end
+
+  def self.log_handle
+    $datawarehouse_log
+  end
+
+  def self.event_monitor_class
+    ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Stream
+  end
+
+  def self.find_target(tags)
+    target = case tags[TAG_TYPE].downcase
+             when "node" then OpenStruct.new(
+               :klass_name     => 'ContainerNode',
+               :find_key       => :name,
+               :value_tag_name => 'nodename',
+             )
+             else
+               $datawarehouse_log.error("unexpected target: [#{tags[TAG_TYPE]}]")
```

Comment:
what value does `target` take then?
add explicit `nil` line (if that's what you meant), or an early return.